src/Readline/Userland.php

```diff
@@ -40,6 +40,8 @@ class Userland implements Readline
      */
     public static function isSupported(): bool
     {
+        static::bootstrapHoa();
```

Comment:
i _think_ `bootstrapHoa` doesn't need to bind terminal resize in this case (because we're not trying to use it, we're just trying to see if we _can_ use it).

maybe we make this optional when bootstrapping?

```
\class_exists('Psy\Readline\Hoa\ConsoleWindow');
```

Reply:
> i _think_ `bootstrapHoa` doesn't need to bind terminal resize in this case (because we're not trying to use it, we're just trying to see if we _can_ use it).
> 
> maybe we make this optional when bootstrapping?
> 
> ```
> \class_exists('Psy\Readline\Hoa\ConsoleWindow');
> ```

Check out my latest changes. Is this what you meant?

Reply:
Yep, thanks!